tests/async/test_async_model_methods.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase
+
+from .models import Book
+
+
+class AsyncModelOperationTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.s1 = Book.objects.create(
+            title="David Copperfield",
+        )
+        cls.s2 = Book.objects.create(
+            title="The Great Gatsby",
+        )
+
+    async def test_asave(self):
+        book = await Book.objects.aget(title="David Copperfield")
+        book.pages = 10
+        await book.asave()
+        getBook = await Book.objects.aget(title="David Copperfield")
+        self.assertEqual(book.pages, getBook.pages)
+
+    async def test_adelete(self):
+        book = await Book.objects.aget(title="The Great Gatsby")
+        await book.adelete()
+        count = await Book.objects.acount()
+        self.assertEqual(count, 1)
```

Comment:
As above, reuse instance from `self.s1`
```suggestion
        await self.s1.adelete()
        count = await SimpleModel.objects.acount()
        self.assertEqual(count, 0)
```